docs/Backend.md

```diff
@@ -0,0 +1,13 @@
+# Backend 
+
+## LINKS
+
+[server folder](/server/)
+
+[api subfolder](/server/api)
+[db subfolder](/server/db)
+
+
+## PACKAGES
+[winston](https://github.com/winstonjs/winston) Saving and displaying logs
+[pg-promise](https://github.com/vitaly-t/pg-promise) ???
```

Comment:
`pg-promise` is a wrapper for simplifying postgres queries. Main functionality it provides are helpers that help create queries for you.